WBooks/src/app/screens/BookDetail/components/DetailBox/styles.js

```diff
@@ -0,0 +1,90 @@
+import { StyleSheet } from 'react-native';
+
+import { white, darkGray, red, green, blue, lightGray, veryBlue } from '../../../../../constants/colors';
+
+const styles = StyleSheet.create({
+  bookContainer: {
+    flexDirection: 'column',
+    paddingTop: 15,
```

Comment:
Check if you can have all paddings in one line. Check padding documentation

Reply:
I tried to do that but I can't find something as simple as the css way. I hope to find something

Reply:
It exists paddingVertical and paddingHorizontal, I use them?

Reply:
I think it will be better